test/topology_random_failures/cluster_events.py

```diff
@@ -558,6 +558,13 @@ async def stop_non_coordinator_node_gracefully(manager: ManagerClient,
     yield
 
 
+@deselect_for(
+    # TODO: remove this skip when #21534 will be resolved.
```

Comment:
My only concern with this patch is: how can we make sure we remember to re-enable this test? If @kostja didn't ping me on this I wouldn't have known that the tests are disabled and wouldn't even know that we need to re-enable them. As my team is responsible for fixing the bug, next time please inform me about it.

Do we, maybe, have some kind of an automatic annotation which would stop skipping this test after the issue is fixed on the branch?

Reply:
Two ways - either the bug mentioned is fixed or better yet, someone has it on their plate, for the coming release, a task to enable it.

As for how we remember... I sometimes ask my wife's help if there's something I need to remember to do. But usually I use my calendar, or the task list.